README.md

```diff
@@ -1,2 +1,28 @@
+
 # stupid-questions
-An app for students to anonymously ask "stupid" questions
+
+This is a small sample project you can use to start a new React/Redux app. It contains:
+
```

Comment:
Can we please update to it is relevant to this project, rather than the starter.